test/parallel/test-process-getactiverequests.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 const fs = require('fs');
 
 for (let i = 0; i < 12; i++)
-  fs.open(__filename, 'r', () => {});
+  fs.open(__filename, 'r', common.mustCall(() => {}));
```

Comment:
In this particular case, the common.mustCall() is not critical because the test itself does not depend in anyway on it.. but, if we're going to have it, because the function is a nonop, it can be omitted...

```suggestion
  fs.open(__filename, 'r', common.mustCall());
```

Reply:
@jasnell, thanks. Followed your suggestion.